src/todo.ts

```diff
@@ -0,0 +1,99 @@
+import { Item } from './item';
+class TodoApp {
+    /**
+     * Allows the creation and maintenance of a simple todo list
+     */
+
+    public toDoList: Map<number, Item>;
+    public curId: number;
+
+    constructor() {
+        this.curId = 1;
+        this.toDoList = new Map<number, Item>();
+    }
+
+    // Had to generate the item ID here so two todoApps could have separate id lists
```

Comment:
Going forward it would be best to use tsdoc syntax for these so IDEs and documentation can be generated.